elodie/result.py

```diff
@@ -8,15 +8,19 @@ def __init__(self):
         self.success = 0
         self.error = 0
         self.error_items = []
+        self.skipped = 0
 
     def append(self, row):
         id, status = row
 
-        if status:
-            self.success += 1
+        if status == 'skipped':
```

Comment:
This is obviously a bit dirty, we might want to introduce something like an Enum here.